modules/install/src/main/scala/coursier/install/GraalvmParams.scala

```diff
@@ -2,7 +2,30 @@ package coursier.install
 
 import dataclass.data
 
+import java.util.Locale
+
 @data class GraalvmParams(
-  defaultVersion: Option[String],
+  defaultVersion: String,
   extraNativeImageOptions: Seq[String]
 )
+
+object GraalvmParams {
+  lazy val defaultGraalvmVersion: String = {
+    val defaultVersion = "22.3.0"
+    if (Platform.isArmArchitecture) s"graalvm-java17:$defaultVersion"
+    else s"graalvm-java11:$defaultVersion"
```

Comment:
Do we really need to use the Java 11 version by default? The Java 17 in both cases should be fine I think.